README.md

```diff
@@ -0,0 +1,40 @@
+# securesystemslib
+
+[![CI](https://github.com/secure-systems-lab/securesystemslib/workflows/Run%20Securesystemslib%20tests/badge.svg)](https://github.com/secure-systems-lab/securesystemslib/actions?query=workflow%3A%22Run+Securesystemslib+tests%22+branch%3Amain)
+
+A cryptography interface to sign and verify
+[TUF](https://theupdateframework.io) and [in-toto](https://in-toto.io)
+metadata.
```

Comment:
This is more honest but maybe goes a bit too far? It's not that you can't sign anything you want...

I've added a bit of hopfully still honest marketing -- also mentioned some tech we use so that the "default installation supports ed25519 only" doesn't scare people away.

```suggestion
Securesystemslib is a cryptography interface for signing and verifying digital signatures. It is developed for the [TUF](https://theupdateframework.io) and [in-toto](https://in-toto.io) projects: the key and signature containers are compatible with metadata formats from those projects.

Under the hood Securesystemslib is able to use various digital signing systems (e.g. [cryptography](https://pypi.org/project/cryptography/), PIV hardware keys and multiple cloud based key management systems)
```